css/css-anchor-position/anchor-position-dynamic-004.html

```diff
@@ -39,9 +39,7 @@
          data-expected-width=30 data-expected-height=20></div>
   </div>
 <script>
-function run_test() {
-  document.body.classList.add('after');
-  checkLayout('.target');
-}
+document.body.classList.add('after');
+checkLayout('.target');
```

Comment:
This brings this test in line with the pattern used in the other `anchor-position-dynamic` tests.